<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editarVendas</class>
 <widget class="QDialog" name="editarVendas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>481</width>
     <height>741</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_1">
    <item>
     <widget class="QLabel" name="Editar_Cliente">
      <property name="font">
       <font>
        <family>Montserrat ExtraBold</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(99, 99, 99);</string>
      </property>
      <property name="text">
       <string>Editar Vendas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cliente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editarVendasCliente">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Produtos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editarVendasProdutos">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Valor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editarVendasValor">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Forma de pagamento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editarVendasPagamento">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Montserrat Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Status Pagamento</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editarVendasStatus">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>790</y>
     <width>211</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cadastrarVenda">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #34D481;
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px
}</string>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelar">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #585858;
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px
}</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelar</sender>
   <signal>toggled(bool)</signal>
   <receiver>editarVendas</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>809</y>
    </hint>
    <hint type="destinationlabel">
     <x>905</x>
     <y>789</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
